files/build_templates/docker_image_ctl.j2

```diff
@@ -108,6 +108,37 @@ function preStartAction()
     fi
 {%- elif docker_container_name == "snmp" %}
     $SONIC_DB_CLI STATE_DB HSET 'DEVICE_METADATA|localhost' chassis_serial_number $(decode-syseeprom -s)
+
```

Comment:
Are you enabling this feature by default for snmp? Shouldn't it be based on configuration?